less/variables.less

```
@@ -11,4 +11,7 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+//Media Query
+@mobile: (max-width: 500px);
```

proper syntax for a media query in Less is 
```suggestion
@mobile: ~"(max-width: 500px)";
```